JustSaying.AwsTools/MessageHandling/SqsPublisher.cs

```diff
@@ -24,7 +24,8 @@ public void Publish(Message message)
             _client.SendMessage(new SendMessageRequest
             {
                 MessageBody = GetMessageInContext(message),
-                QueueUrl = Url
+                QueueUrl = Url,
+                DelaySeconds = message.DelaySeconds.GetValueOrDefault()
```

Comment:
`message.DelaySeconds.GetValueOrDefault()` is equivalent to `message.DelaySeconds ?? 0`